frontend/src/pages/index.page.tsx

```diff
@@ -249,6 +251,14 @@ const Home: NextPage = () => {
             />
           </div>
         </section>
+
+        {isFlagActive(runtimeData.data, "intro_pricing_countdown") &&
```

Comment:
I think you used the wrong flag here :)

```suggestion
        {isFlagActive(runtimeData.data, "bundle") &&
```

Reply:
Oops, I must've forgotten to change it.